torch/ao/sparsity/experimental/data_sparsifier/lightning/callbacks/data_sparsity.py

```
@@ -0,0 +1,59 @@
+from copy import deepcopy
+import torch
+from dataclasses import dataclass, field
+from typing import Any, Optional, Dict
+import pytorch_lightning as pl
```

```
import pytorch_lightning as pl  # type: ignore
```

---

This still seems to throw TYPEIGNORE error. I get this 
```This line has an unqualified `type: ignore`; please convert it to `type:
    ignore[xxxx]```

---

```type: ignore[import]``` seemed to fix the issue.